packages/lesmis/src/components/LocationHeader.js

```diff
@@ -116,9 +131,21 @@ export const LocationHeader = () => {
           <FlexStart>
             {/* Todo: add exports @see */}
             {/* https://app.zenhub.com/workspaces/active-sprints-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/2511 */}
-            <IconButton startIcon={<GetApp />}>
-              <I18n t="dashboards.export" />
-            </IconButton>
+            {view === 'map' ? (
+              <MapTableModal
+                title={exportTitle}
+                overlayReportData={overlayReportData}
+                Button={props => (
+                  <IconButton {...props} startIcon={<AssignmentIcon />}>
+                    <I18n t="dashboards.export" />
+                  </IconButton>
+                )}
+              />
+            ) : (
+              <IconButton startIcon={<GetApp />}>
```

Comment:
Now that we're close to releasing it without including full dashboard export, we should probably hide this button that does nothing right @enunan?